test/run_envoy_bazel_coverage.sh

```diff
@@ -20,10 +20,14 @@ elif [[ -n "${COVERAGE_TARGET}" ]]; then
   COVERAGE_TARGETS=${COVERAGE_TARGET}
 else
   # For fuzz builds, this overrides to just fuzz targets.
-  COVERAGE_TARGETS=//test/... && [[ ${FUZZ_COVERAGE} == "true" ]] &&
-    COVERAGE_TARGETS="$(bazel query 'attr("tags", "fuzz_target", //test/...)')"
+  COVERAGE_TARGETS=//test/...
 fi
 
+if [[ "${FUZZ_COVERAGE}" == "true" ]]; then
```

Comment:
This if block has the same condition as the next one.  Can they be merged?

Reply:
Good point. Done. They were two difference "steps", so logically seperated
(filter targets, add build options)
Added a comment explaining.